linter/internal/types/build_graph.go

```diff
@@ -217,6 +217,8 @@ func calcTP(node ast.Node, varAt map[ast.Node]*common.Variable, g *typeGraph) ty
 		return tpRef(g.getExprPlaceholder(imported))
 	case *ast.ImportStr:
 		return tpRef(stringType)
+	case *ast.ImportBin:
+		return tpRef(anyArrayType)
```

Comment:
I wasn't confident on this line - is this correct?  Is there a better value to return?

Reply:
It's correct, but it would be better to return an array of numbers specifically. We can add another hardcoded type for this (alongside anyArrayType):
* Here you can add a new type id: https://github.com/google/go-jsonnet/blob/74b3a1b66972ebc132fc3f3c2778ebc4b76b931b/linter/internal/types/placeholder.go#L15
* Here you actually define it: https://github.com/google/go-jsonnet/blob/74b3a1b66972ebc132fc3f3c2778ebc4b76b931b/linter/internal/types/graph.go#L131
* Here's another place which could benefit from it: https://github.com/google/go-jsonnet/blob/cb9061e99e6769ccc95c697acd8354370efc113d/linter/internal/types/stdlib.go#L9

Reply:
Thanks for the hints.  Added `numberArrayType` and a testcase.